components/palette.css

```diff
@@ -51,16 +51,11 @@
 .textDanger   { color: var(--textDangerColor) }
 .textMuted    { color: var(--textMutedColor) }
 
+/* TODO - needs to be refactored since this file shouldn't just be a place to stuff classes */
```

Comment:
the general pattern I'm following is that any class that's reused in multiple components goes here, since most/all of the .css files should import palette already.  If we move them somewhere else, we'll have to import that file as well.